beacon-chain/monitor/process_exit.go

```diff
@@ -0,0 +1,31 @@
+package monitor
+
+import (
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/block"
+	"github.com/sirupsen/logrus"
+)
+
+// processExitsFromBlock logs the event of one of our tracked validators' exit was
+// included in a block
+func (s *Service) processExitsFromBlock(blk block.BeaconBlock) {
```

Comment:
I think i will like cleaner abstraction. By renaming this function to `processExits` and just take in `exits []*ethpb.SignedVoluntaryExit` as argument

Reply:
The problem is that we need to report in which block this was included, so either we take also the slot, or we take the block as I am taking here.

Reply:
Ah. Block is fine then